lib/src/conflicts.rs

```diff
@@ -188,6 +190,7 @@ pub struct MaterializedFileConflictValue {
     /// Merged executable bit. `None` if there are changes in both executable
     /// bit and file absence.
     pub executable: Option<bool>,
+    pub copy_id: CopyId,
```

Comment:
Shouldn't this be `Option<CopyId>`? iirc, the idea was that `jj resolve` errors out if `executable` or `copy_id` can't be resolved.